### DISCUSSION TITLE
Can you use defineNuxtLink with `noPreFetch`?

### DISCUSSION THREAD

**Discussion by matthew-dean** [nuxt/nuxt#23770] [Questions]
There's instructions here for defining a nuxt link: https://nuxt.com/docs/api/components/nuxt-link#definenuxtlink-signature

However, in the signature, it doesn't list `noPreFetch`, and it doesn't appear to work. I was trying to use this to avoid prefetching site-wide, and avoid [some of the problems with prefetching in Nuxt](https://github.com/nuxt/nuxt/issues/22817).

**Comment by matthew-dean** [comment]
😢

**Comment by TheAlexLichter** [comment]
The easiest option is creating an own Wrapper for now.

But IMO it would be reasonable request. Would you mind creating an issue @matthew-dean?
